Source/Core/Core/HW/WiimoteEmu/Extension/Classic.cpp

```diff
@@ -78,9 +78,8 @@ Classic::Classic() : Extension1stParty("Classic", _trans("Classic Controller"))
   groups.emplace_back(m_buttons = new ControllerEmu::Buttons(_trans("Buttons")));
   for (const char* button_name : classic_button_names)
   {
-    const std::string& ui_name = (button_name == std::string("Home")) ? "HOME" : button_name;
-    m_buttons->controls.emplace_back(
-        new ControllerEmu::Input(ControllerEmu::DoNotTranslate, button_name, ui_name));
+    std::string ui_name = (button_name == std::string("Home")) ? "HOME" : button_name;
```

Comment:
```suggestion
    std::string ui_name = (button_name == std::string_view("Home")) ? "HOME" : button_name;
```

Reply:
Actually, it might be worth making `classic_button_names` an array of string views to avoid having to calculate the string lengths all the time. Doing that also has the benefit of making `button_name == "Home"` just work.

Reply:
Made it an array of string_view.